### PR TITLE
[PM-2241] Add PRF attestation flow during passkey registration

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by jlf0dev — COMMENTED** [review]
Fantastic work, I love all the comments and improvements to naming. I found a small issue and had a minor question.

File: apps/web/src/app/auth/core/services/webauthn-login/webauthn-login-admin.service.ts [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+import { RotateableKeySetService } from "../rotateable-key-set.service";
 
 import { SaveCredentialRequest } from "./request/save-credential.request";
 import { WebauthnLoginAttestationResponseRequest } from "./request/webauthn-login-attestation-response.request";
+import { createSymmetricKeyFromPrf, getLoginWithPrfSalt } from "./utils";
 import { WebauthnLoginApiService } from "./webauthn-login-api.service";
 
 @Injectable({ providedIn: "root" })
-export class WebauthnLoginService {
+export class WebauthnLoginAdminService {
```
[the remark this answers is not in the document]
Comment (coroiu) [reply apps/web/src/app/auth/core/services/webauthn-login/webauthn-login-admin.service.ts:20]: @JaredSnider-Bitwarden is gonna be introducing `WebauthnLoginService` as part of [PM-2041 Finish work to add FIDO2 credential authentication to web client login component](https://bitwarden.atlassian.net/browse/PM-2041) so I'm renaming this in anticipation of that :)

File: apps/web/src/app/auth/core/services/webauthn-login/request/save-credential.request.ts [hunk 2]
```diff
@@ -4,7 +4,10 @@ import { WebauthnLoginAttestationResponseRequest } from "./webauthn-login-attest
  * Request sent to the server to save a newly created webauthn login credential.
  */
 export class SaveCredentialRequest {
-  /** The response recieved from the authenticator. This contains the public key */
+  /**
+   * The response recieved from the authenticator.
```
[the remark this answers is not in the document]
Comment (coroiu) [reply apps/web/src/app/auth/core/services/webauthn-login/request/save-credential.request.ts]: Fixed!

**Review by jlf0dev — APPROVED** [review]

**Review by justindbaur — APPROVED** [review]
Platform changes look good, thanks for making it be it's own type!

**Review by justindbaur — APPROVED** [review]

**Review by jlf0dev — APPROVED** [review]

**Pull request merged**